src/admin/tables/task.php

```diff
@@ -0,0 +1,283 @@
+<?php
+/**
+ * @package    Com_Admintask
+ * @author     __COMPONENT_CONTACT__
+ * @copyright  __COMPONENT_COPYRIGHT__
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+// No direct access
+defined('_JEXEC') or die;
+
+use Joomla\Utilities\ArrayHelper;
+/**
+ * task Table class
+ *
+ * @since  1.0
+ */
+class AdmintaskTabletask extends JTable
+{
+	/**
+	 * Constructor
+	 *
+	 * @param   JDatabase  &$db  A database connector object
+	 */
+	public function __construct(&$db)
+	{
+		JObserverMapper::addObserverClassToClass('JTableObserverContenthistory', 'AdmintaskTabletask', array('typeAlias' => 'com_admintask.task'));
+		parent::__construct('#__admintask_tasks', 'id', $db);
+	}
+
+	/**
+	 * Overloaded bind function to pre-process the params.
+	 *
+	 * @param   array  $array   Named array
+	 * @param   mixed  $ignore  Optional array or list of parameters to ignore
+	 *
+	 * @return  null|string  null is operation was satisfactory, otherwise returns an error
+	 *
+	 * @see     JTable:bind
+	 * @since   1.5
+	 */
+	public function bind($array, $ignore = '')
```

Comment:
This doesn't need at all. Can you explain what you have changed in this method and what is new in this method which is not in it's parent method?